dashboard/test/controllers/javabuilder_sessions_controller_test.rb

```diff
@@ -56,6 +56,17 @@ class JavabuilderSessionsControllerTest < ActionController::TestCase
     assert_response :success
   end
 
+  test 'student of csa pilot participant can get access token' do
+    teacher = create(:teacher)
+    create(:single_user_experiment, min_user_id: teacher.id, name: 'csa-pilot')
```

Comment:
Great test! Might also be worth adding a test to check that if the user is not enrolled under a teacher that has the CSA pilot that they wouldn't have access?

It would also be great to define `csa-pilot` as a constant somewhere and reference that const instead of the string directly in all the places, (less room for error/misspelling)

Reply:
There does exist a test for a student with a non-csa-pilot teacher already, but I did just add one for a student of a pilot teacher where the pilot isn't the csa-pilot. Also defined these constants!